app/src/services/feature-flags/setup.ts

```diff
@@ -0,0 +1,10 @@
+import { FeatureFlagManager } from "./FeatureFlagManager";
+import { LocalFeatureFlagManager } from "./LocalFeatureFlagManager";
+
+export interface FlagManager {
+  isFeatureEnabled(feature: string, userId?: string): Promise<boolean>;
+}
+
+export const manager: FlagManager = process.env.FEATURE_FLAGS_PROJECT
```

Comment:
I prefer not to do environment checks in our source code since that breaks 12 factor app principles. A couple of reasons: (1) if the logic in the code has branches based on environment variables, it means that we have less confidence that the code branch that's covered in one environment (e.g. automated test suite) would work the same as the code branch that's covered when deployed to production and. Relatedly, in terms of test coverage tools, one entire branch of code will not be covered. (2) it makes it harder to implement automated integration tests if the test suite is forced to not use the AWS flag manager.

If we want to be more explicit about when we're using one flag manager type over the other without having to comment out or remove the FEATURE_FLAGS_PROJECT env var, we could always add an extra env var like FEATURE_FLAG_MANAGER_TYPE that is either "aws" or "local" e.g.

```
manager = process.env.FEATURE_FLAG_MANAGER_TYPE === "aws" ? new AWSFeatureFlagManager(process.env.FEATURE_FLAGS_PROJECT) : new LocalFeatureFlagManager();
```

would that address the issue you mentioned without a NODE_ENV check?

Reply:
Cross-posting from Slack:

Sorry, I caused confusion with an inaccurate statement — Next.js only loads env vars into the test environment from the .env file, but it doesn’t load the vars in files like .env.development or .env.local so I think ultimately 
Ali. your existing condition that just checks the FF project var would be enough.